x-pack/packages/kbn-cloud-security-posture-common/schema/vulnerabilities/csp_vulnerability_finding.ts

```diff
@@ -7,7 +7,7 @@
 
 // TODO: this needs to be defined in a versioned schema
 import type { EcsEvent } from '@elastic/ecs';
-import { VulnSeverity } from '../types_old';
+import type { VulnSeverity } from '../..';
```

Comment:
I think it's better to import directly from `../../types/vulnerabilities` than from index where it is re-exported